tern/formats/cyclonedx/cyclonedx.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+from tern.classes.template import Template
+
+
+class CycloneDX(Template):
```

Comment:
The template is meant to provide a mapping from Tern's metadata field names to CycloneDX's field names. It looks like whatever property names Tern uses, CycloneDX uses (`name` and `version`). So you don't need this file.